### DISCUSSION TITLE
Fatal error when starting

### DISCUSSION THREAD

**Discussion by Pnoexz** [Tyrrrz/LightBulb#281] [General]
![image](https://github.com/Tyrrrz/LightBulb/assets/3297901/dd8cf3f9-61f1-4aa1-9607-a536ac3a439d)

This happens after the main UI loads (nothing can be clicked, tho) but before the color is changed. Can't see the application settings, but it should be on the latest version unless it's an issue with the updater. This didn't happen yesterday

**Comment by Tyrrrz** [comment]
Yeah, I just noticed that bug too, during auto-update. The newest release should have this fixed, but you will need to manually download it, unfortunately. https://github.com/Tyrrrz/LightBulb/releases/tag/2.4.10

**Comment by Pnoexz** [comment]
Thanks for the quick patch and response. It works properly after manually downloading it again

**Reply by Tyrrrz** [reply]
Thanks and sorry for the inconvenience
